src/components/PersonalSettings.vue

```diff
@@ -34,6 +34,17 @@
 			</em>
 		</p>
 
+		<!-- user settings iframe  -->
+		<div id="user-cool-frame-section" class="section">
+			<h2>{{ t('richdocuments', 'Collabora User Settings') }}</h2>
```

Comment:
We should call this differently as we want to remove the duplicate word "settings" from all seections we have (ref https://github.com/nextcloud/richdocuments/issues/4351)

Reply:
How it looks now: 
![nextcloud local_index php_settings_user_richdocuments (1) (1)](https://github.com/user-attachments/assets/24a3313b-92b0-473c-b7d8-f703491ac594)


I have discussed with Vivek that we should also remove `Collabora` since it's part of Office settings section.

If we also remove settings (even thought I do see "Personal settings" in the https://github.com/nextcloud/richdocuments/issues/4351 you have referenced) then we either replace it with something such as "preferences" or - and I think better - we simply remove that h2 completely and let the headings in the iframe being h2 (so, autotext and workbook will become upper level which probably makes more sense)

Reply:
@pedropintosilva Thanks for adding a screenshot :)